src/modules/dashboard/components/Dialogs/AddExistingSafeDialog/AddExistingSafeDialogForm.tsx

```diff
@@ -197,7 +228,12 @@ const AddExistingSafeDialogForm = ({
   };
 
   useEffect(() => {
-    if (!isLoadingSafe && (safeData === undefined || safeData === null)) {
+    if (
+      !isLoadingSafe &&
+      (safeData === undefined ||
+        safeData === null ||
+        safeData === 'alreadyExists')
```

Comment:
'alreadyExists' is used many places in the file, maybe a constant could be an alternative?

Reply:
@danbr Good suggestion, I've implemented.